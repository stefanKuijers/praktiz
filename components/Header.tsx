// import Link from 'next/link';

import React from 'react';
import { createStyles, makeStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import Container from '@material-ui/core/Container';

const useStyles = makeStyles(() =>
    createStyles({
        root: {
            flexGrow: 1,
        },
        title: {
            flexGrow: 1,
        },
    }),
);

const Header: React.FC = () => {
    const classes = useStyles();

    return (
        <div className={classes.root}>
            <AppBar position="static">
                <Toolbar>
                    <Container>
                        <Typography variant="h6" className={classes.title}>
                            Praktiz
                        </Typography>
                    </Container>
                </Toolbar>
            </AppBar>
        </div>
    );
};

export default Header;
