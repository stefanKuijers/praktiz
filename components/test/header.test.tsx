import { render, RenderResult } from '@testing-library/react';
import React from 'react';

import Header from '../Header';

const setup = (component: React.ReactElement): RenderResult => {
    const tree = render(component);

    return {
        ...tree,
    };
};

describe('<Header />', () => {
    it('matches snapshot', async () => {
        const { container } = setup(<Header />);
        expect(container).toMatchSnapshot();
    });
});
