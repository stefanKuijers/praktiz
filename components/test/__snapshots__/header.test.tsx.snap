// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> matches snapshot 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg"
        >
          <h6
            class="MuiTypography-root makeStyles-title-2 MuiTypography-h6"
          >
            Air Q
          </h6>
        </div>
      </div>
    </header>
  </div>
</div>
`;
